src/app/fsat/fsat-tabs/fsat-tabs.component.ts

```diff
@@ -51,6 +52,18 @@ export class FsatTabsComponent implements OnInit {
   modifyConditionsTabSub: Subscription;
   modifyConditionsTab: string;
 
+  stepTabs: Array<string> = [
```

Comment:
Let's remove stepTabs array here and in fsat.component and make it a property of fsat.service